.github/workflows/fw_icu4c_ci.yml

```diff
@@ -77,3 +108,18 @@ jobs:
           cd test/cintltst;
           echo 'Running ICU4C cintltst';
           make -j2 check;
+
+  debian-packaging:
+    needs: set-version-number
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checkout ci-build-tools
+        run: git clone https://github.com/sillsdev/ci-builder-scripts
+
+      - name: Build debian package
+        run: |
+          sudo apt-get install -y devscripts dh-make
+          cd icu4c/
+          ../ci-builder-scripts/bash/make-source --dists "focal bionic" --arches "amd64 i386" --main-package-name "icu-fw" --supported-distros "focal bionic" --debkeyid "TODO-secrets.ICU_DEBKEYID" --main-repo-dir . --package-version ${{needs.set-version-number.outputs.version}} --build-in-place
```

Comment:
I think this is a non-issue since the version is based off the ICU version number, and the GHA build number only. Any beta or such information is only generated by the packages made during the PR. I may want to set a beta version for the purposes of the PR artifacts, I don't need one for the Nuget packages because they are going to be tested by installing them locally in a dev machine. For the linux side I'm not sure if we can test in a similar fashion, so making ~beta packages to verify with may be good.